admin/bootstrap.py

```diff
@@ -75,6 +76,34 @@ def is_tails():
     return id == 'Tails'
 
 
+def clean_up_tails3_venv(VENV_DIR):
```

Comment:
heads up best to replace `def clean_up_tails3_venv(VENV_DIR)` with `def clean_up_tails3_venv(virtualenv_path=VENV_DIR)` and use `virtualenv_path` within the function. otherwise there are going to be two `VENV_DIR` variables with different scopes (one global, one local) - this _will_ work as python's scope resolution will select the local first but this could cause confusion in the future